app/javascript/src/Navigation/components/ContextSelector.jsx

```diff
@@ -44,18 +44,33 @@ class ContextSelector extends React.Component {
     )
   }
 
-  getClassNamesFor ({ menu, api }) {
-    const { activeMenu, currentApi } = this.props
+  getClassNamesForMenu (menu) {
+    const { activeMenu } = this.props
 
     if (menu === 'dashboard' && activeMenu === 'dashboard' ||
-      menu === 'audience' && (['buyers', 'finance', 'cms', 'site'].indexOf(activeMenu) !== -1) ||
-      api && (['serviceadmin', 'monitoring'].indexOf(activeMenu) !== -1) && api === currentApi.service.id) {
+      menu === 'audience' && (['buyers', 'finance', 'cms', 'site'].indexOf(activeMenu) !== -1)) {
```

Comment:
Maybe extract to an array
`const menuNames = ['buyers', 'finance', 'cms', 'site'];`

Reply:
Good idea! This actually gives me some ideas for a better refactoring, I will address it in a different PR.